### PR TITLE
Fixed bison command in makefile

### PR DESCRIPTION
**Pull request by absolutelynothinghere**
The bison command was missing `-b awkgram`

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -37,7 +37,7 @@ CC = $(HOSTCC)  # change this is cross-compiling.
 # By fiat, to make our lives easier, yacc is now defined to be bison.
 # If you want something else, you're on your own.
 # YACC = yacc -d -b awkgram
-YACC = bison -d
+YACC = bison -d -b awkgram
```
Comment (DIEGOHJJACOBO) [inline makefile:40]:
```suggestion
YACC = bison -d -b awkgram
```

**Review by DIEGOHJJACOBO — APPROVED** [review]

**Comment by arnoldrobbins** [comment]
What is the point of this change? Does awk not build for you? If so, on what system? The current makefile works just fine on at least Linux and MacOS.  Maybe you need a newer version of bison?  Thanks.

**Comment by absolutelynothinghere** [comment]
yacc implementations *other than bison* need this change, see also https://github.com/onetrueawk/awk/issues/204

Is there harm in pushing this change? Does it stop awk from compiling on your machine?

**Comment by plan9** [comment]
we already accommodated yaccs not named bison with the appropriate arguments for filename prefix.

**Comment by absolutelynothinghere** [comment]
Can you link to the aforementioned accommodations? I don't see any changes in the Makefile.

**Pull request closed without merging**